Instagram/View Controllers/DetailsViewController.m

```diff
@@ -43,6 +46,9 @@ - (void)viewDidLoad {
     NSString *likeCountStr = [NSString stringWithFormat:@"Liked by %@ people", self.post.likeCount];
     [self.likeLabel setText:likeCountStr];
     
+    NSString *commentCountStr = [NSString stringWithFormat:@"View all %@ comments", self.post.commentCount];
```

Comment:
Objective C convention is to not use abbreviations. It's better to write `commentCountString`.